dags/bqetl_mobile_search.py

```diff
@@ -29,6 +29,7 @@
         email=["bewu@mozilla.com", "telemetry-alerts@mozilla.com"],
         date_partition_parameter="submission_date",
         depends_on_past=False,
+        allow_field_addition_on_date=2020 - 07 - 13,
```

Comment:
Looks like we need to pick a different formatter here.

Reply:
it was missing quotes, so black added spaces. fixed.